.nxignore

```diff
@@ -10,3 +10,5 @@ packages/fluentui/README.md
 packages/fluentui/.browserslistrc
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32359&commitId=2efef2b4328b178507cab8af833a26981191f400&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|1| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|3| Changed|
|Avatar Converged.size+image+badge.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>